client/src/utils/parser/robotCodeToSsotParsing/robotCodeToSsotParsing.js

```diff
@@ -0,0 +1,369 @@
+
+/**
+ * @category Client
+ * @module
+ */
+
+const { default: customNotification } = require("../../notificationUtils");
+
+const FOURSPACE = '    ';
+
+
+/**
+ * 
+ * @returns "uniqueId" which is just an increment from the counter in the local storage 
+ */
+const getUniqueId = () => {
+    const newId = JSON.parse(sessionStorage.getItem('idCounter')) + 1;
+    sessionStorage.setItem('idCounter', newId);
+    return newId;
+}
+
+/**
+ * #todo
```

Comment:
Please address this todo

Reply:
uff... just forgot that... Thanks => it's documented now